frontend/packages/shared/src/components/altinnHeaderMenu/AltinnHeaderMenu.tsx

```diff
@@ -0,0 +1,36 @@
+import { getTopBarMenu } from 'app-development/layout/AppBar/appBarConfig';
+import { getRepositoryType } from 'app-shared/utils/repository';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link, useParams } from 'react-router-dom';
+import classes from './AltinnHeaderMenu.module.css';
+import classNames from 'classnames';
+
+export interface IAltinnHeaderMenuProps {
+  activeSubHeaderSelection?: string;
+}
+
+export const AltinnHeaderMenu = ({ activeSubHeaderSelection }: IAltinnHeaderMenuProps) => {
+  const { t } = useTranslation();
+  const { org, app } = useParams();
+  const repositoryType = getRepositoryType(org, app);
+  const menu = getTopBarMenu(repositoryType);
```

Comment:
This is still app-development specific. This menu component should take the menu items in as props from the outside, so that we can use the header in a more generic context.
Suggest adding `menu` as a prop to this component, and then passing it in.

Reply:
Added menu as props